x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/settings/assistant_settings.tsx

```diff
@@ -70,131 +57,101 @@ export const AssistantSettings: React.FC<Props> = React.memo(
     defaultConnector,
     onClose,
     onSave,
-    selectedConversationId: defaultSelectedConversationId,
-    onConversationSelected,
     conversations,
     conversationsLoaded,
+    setPaginationObserver,
   }) => {
-    const { http, toasts, selectedSettingsTab, setSelectedSettingsTab } = useAssistantContext();
+    const {
+      currentAppId,
+      http,
+      assistantAvailability: { isAssistantEnabled },
+      selectedSettingsTab,
+      setSelectedSettingsTab,
+      toasts,
+    } = useAssistantContext();
 
     useEffect(() => {
       if (selectedSettingsTab == null) {
-        setSelectedSettingsTab(CONVERSATIONS_TAB);
+        setSelectedSettingsTab(QUICK_PROMPTS_TAB);
```

Comment:
Why default to quick prompts tab vs conversations if there was no previous selection?

Reply:
Because we never show conversations in the modal. There was one way to see it and James approved removing it. Now the only views available in the modal are prompt settings (quick and system).

The only previous path to get to the conversations modal was this `Conversation Settings` link:
<img width="679" alt="Screenshot 2025-02-20 at 2 29 05 PM" src="https://github.com/user-attachments/assets/bf0c8566-697d-4bd4-8205-cd3402858d84" />
Now removed:
<img width="633" alt="Screenshot 2025-02-25 at 3 53 54 PM" src="https://github.com/user-attachments/assets/8aa8aab2-69e4-41d0-9806-b0d62b3f8b75" />

Reply:
Ahhh, that makes sense! 👍 

Side note: I think we can finally get rid of the model now that the dedicated settings can be viewed in serverless, no? I will bring this up in the weekly sync.

Reply:
Yeah, I think some of the design decision was to go back to isolated dedicated UI's where needed for 'quick settings' and rely on the main assistant settings for everything else. That said, so long as there aren't any RBAC issues to getting to the main assistant settings, we should be able to remove the old modal now 🤞